tests/test_deterministic.py

```diff
@@ -15,7 +16,9 @@ def test_deterministic_training_common(algo):
     kwargs = {"policy_kwargs": dict(net_arch=[64])}
     env_id = "Pendulum-v1"
     if algo in [TD3, SAC]:
-        kwargs.update({"action_noise": NormalActionNoise(0.0, 0.1), "learning_starts": 100, "train_freq": 4})
+        kwargs.update(
+            {"action_noise": NormalActionNoise(np.zeros(1), 0.1 * np.ones(1)), "learning_starts": 100, "train_freq": 4}
```

Comment:
we need to check in the doc that we always use a numpy array as input, otherwise, it might break some codes

Reply:
I did a check on the docs and found numpy arrays used in instances where `NormalActionNoise` is called. For now, it should be fine. 

In the type hints, it's also `np.ndarray` expected for both parameters. Since `np.random.normal` accepts both floats and numpy arrays, the current `__call__` will run regardless.
https://github.com/DLR-RM/stable-baselines3/blob/bea3c44ba52278ec755af0179859b04ab80cdcaf/stable_baselines3/common/noise.py#L27-L41

With the introduction of `.astype`, code that uses floats will fail; perhaps another PR to cast python floats to numpy arrays?

Reply:
> With the introduction of .astype, code that uses floats will fail; perhaps another PR to cast python floats to numpy arrays?

should be fine if we don't have example in the doc, and the type annotations were corrects.
It might break some code but should be very limited (1D action space problems).